topics/chip-seq/tutorials/formation_of_super-structures_on_xi/tutorial.md

```diff
@@ -0,0 +1,862 @@
+>    > >
+>    > {: .solution }
+>    {: .question}
+>
+{: .hands_on}
+
+It is often necessary to trim sequenced read, for example, to get rid of bases that were sequenced with high uncertainty (= low quality bases).
+
+> ### {% icon question %} Questions
+>
+> What is the relation between `wt_H3K4me3_read1` and `wt_H3K4me3_read2`?
+>
+> > ### {% icon solution %} Solution
+> > The ChIP-seq data for `wt_H3K4me3_rep1` has been sequenced using paired-end sequencing. 
+> >
+> > The paired-end sequencing is based on the idea that the initial DNA fragments (longer than the actual read length) is sequenced on both its ends. This approach results in two reads per fragment, with the first read in forward orientation and the second read in reverse-complement orientation.
+> > 
+> > With paired-end sequencing, more of each fragment is covered than with single-end sequencing (only forward orientation sequenced), given that the read length is the same but twice more reads are generated for the same fragments
+> >
+> > ```
+> >     ------>
+> >       ------>
+> >         ------>
+> >           ------>
+> >
+> >     ----------------------------- [fragment]
+> >
+> >     ------>         <------
+> >       ------>         <------
+> >         ------>         <------
+> >           ------>         <------
+> > ```
+> > 
+> > The paired-end sequencing generates then 2 files:
+> > - One file with the sequences corresponding to foward orientation of all the fragments
+> > - One file with the sequences corresponding to reverse orientation of all the fragments
+> >
+> > Here `wt_H3K4me3_read1` corresponds to the forward reads and `wt_H3K4me3_read2` to the reverse reads.
+> {: .solution }
+{: .question}
+
+> ### {% icon hands_on %} Hands-on: Quality control
+>
+> 1. Run **Trim Galore!** {% icon tool %} with the following parameters
+>    - *"Is this library paired- or single-end?"*: `Paired-end`
+>       - {% icon param-file %} *"Reads in FASTQ format"*: `wt_H3K4me3_read1` (Input dataset)
+>       - {% icon param-file %} *"Reads in FASTQ format"*: `wt_H3K4me3_read2` (Input dataset)
+> 
+>       The order is important here!
+>
+>       > ### {% icon tip %} Tip: Not selectable files?
+>       >
+>       > If your FASTQ files cannot be selected, you might check whether their format is FASTQ with Sanger-scaled quality values (`fastqsanger`). You can edit the data type by clicking on the `pencil` symbol.
+>      {: .tip}
+>
+>    - *"Trim Galore! advanced settings"*: `Full parameter list`
+>       - *"Trim low-quality ends from reads in addition to adapter removal"*: `15`
+>       - *"Overlap with adapter sequence required to trim a sequence"*: `3`
+>       - *"Generate a report file"*: `Yes`
+>
+> 2. Inspect the generated txt file (`report file`)
+>
+>    > ### {% icon question %} Questions
+>    >
+>    > 1. In the forward reads, how many bp has been remove because of bad quality?
```

Comment:
How many basepairs has been removed from the forwards reads because of bad quality?